modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/dao/search/ContentDashboardItemSearchContainerFactory.java

```diff
@@ -93,14 +96,39 @@ public static ContentDashboardItemSearchContainerFactory getInstance(
 		return searchContainer;
 	}
 
+	public SearchContainer<ContentDashboardItem<?>> createWithAllResults()
+		throws PortletException {
+
+		SearchContainer<ContentDashboardItem<?>> searchContainer =
+			new SearchContainer<>(
+				_portletRequest,
+				PortletURLUtil.clone(
+					PortletURLUtil.getCurrent(
+						PortalUtil.getLiferayPortletRequest(_portletRequest),
+						PortalUtil.getLiferayPortletResponse(_portletResponse)),
+					PortalUtil.getLiferayPortletResponse(_portletResponse)),
+				null, "there-is-no-content");
+
+		searchContainer.setOrderByCol(_getOrderByCol());
+		searchContainer.setOrderByType(_getOrderByType());
+
+		SearchResponse searchResponse = _getSearchResponse(-1, -1);
```

Comment:
Instead of duplicating the whole method just to change 1 line, can we refactor so that we keep the common code unique?